tests/components/plugwise/test_sensor.py

```
@@ -3,41 +3,28 @@
 from unittest.mock import MagicMock
 
 import pytest
+from syrupy.assertion import SnapshotAssertion
 
 from homeassistant.components.plugwise.const import DOMAIN
 from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
 from homeassistant.helpers.entity_component import async_update_entity
```

Ruff (0.12.2)

`homeassistant.helpers.entity_component.async_update_entity` imported but unused

Remove unused import: `homeassistant.helpers.entity_component.async_update_entity`

(F401)